bin/package-requires-update

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env node
+
+var exec = require( 'child-process-promise' ).exec;
+var packageJson = require( '../' + process.argv[2] );
+var semver = require( 'semver' );
+
+if ( packageJson.private ) {
+	process.exitCode = 0;
+}
+
+if ( /[^A-Za-z0-9_\/:=-]/.test( packageJson.name ) ) {
+	console.log( packageJson.name, 'Package name cannot contain spaces and/or invalid characters' );
+	console.log( '\n' );
+	process.exitCode = 1;
+} else {
+	exec( 'npm view ' + packageJson.name + ' version' )
+		.then( function( result ) {
+			if ( semver.gt( packageJson.version, result.stdout ) ) {
+				process.exitCode = 0;
+			} else {
+				console.log( packageJson.name, 'Package must be versioned' );
```

Comment:
I think he just meant that the `package.json` file must have a `version` attribute set. I think it makes sense to return an error if it's not set here since we won't be able to compare it to the last published version in `bin/publish-modules`.